tests/hpc/slurm_master.pm

```diff
@@ -272,6 +276,9 @@ sub run {
     if ($slurm_conf =~ /nfs_db/) {
         %test = accounting_test_01();
         pars_results(%test);
+    } elsif ($slurm_conf =~ /accounting/) {
```

Comment:
this could be added to the condition above, as you are doing the exact same thing here

Reply:
This will differ soon, as there are more changes for this part coming. If you like I could change it, but if it is acceptable, I would keep this in such way.